### ISSUE TITLE
Blur on Enter key press caught by submit button

### ISSUE THREAD

**Issue by amangeot** [erikras/redux-form#3139]
Hello,
I'm not sure if this is intended or if this is a bug but blur is not triggered when Enter key is pressed and caught by form's submit button.

---
I currently have a form with a few fields, and a submit button.

When I focus a field, then click the submit button, here are the actions triggered:
![image](https://user-images.githubusercontent.com/7119159/27740455-60d66ede-5db2-11e7-8250-7e6161513e69.png)

If I focus a field then press Enter key, the following actions are triggered:
![image](https://user-images.githubusercontent.com/7119159/27740562-c0fc21d2-5db2-11e7-88f4-1134ecaf80f0.png)

Redux-form: v6.8.0

---
If this is an intended behaviour, what's the best way to handle blur on keyboard events ?

Thank you,

**Comment by gustavohenke** [comment]
I don't think there's anything you can do about it other than manually blurring/dispatching `blur` on that field.

The reason is that the browser itself doesn't dispatch the `blur` event when you press `enter`: 
https://jsfiddle.net/gustavohenke/se9d49s7/

**Comment by amangeot** [comment]
Oh ok thank you for the precision. Just in case it could be useful for someone else, I went for a blur on active element inside the `onSubmit` handler:
```
// NOTE: Blur active element to trigger a blur on Enter key press
if (document && document.activeElement) document.activeElement.blur()
```

**Comment by amangeot** [comment]
It was actually fixed in material-ui v0.18.6: https://github.com/callemall/material-ui/pull/7283
